packages/app-mobile/components/NoteBodyViewer/bundledJs/noteBodyViewerBundle.ts

```diff
@@ -1,17 +1,17 @@
 
 import WebViewToRNMessenger from '../../../utils/ipc/WebViewToRNMessenger';
-import { NoteViewerLocalApi, NoteViewerRemoteApi, RendererWebViewOptions } from './types';
+import { NoteViewerLocalApi, NoteViewerRemoteApi, RendererWebViewOptions, WebViewLib } from './types';
 import Renderer from './Renderer';
 
 declare global {
 	interface Window {
 		rendererWebViewOptions: RendererWebViewOptions;
-		webviewLib: { postMessage: (message: string)=> void };
+		webviewLib: WebViewLib;
```

Comment:
Given that all commits from Joplin pull requests are squashed, I'm not certain that a pull request is the best way to complete this merge. As such, I'm closing this pull request for now.

The change highlighted above should be the only change necessary to resolve the merge conflict.